Data-Visualization/BarChart/bar_chart.py

```diff
@@ -0,0 +1,34 @@
+import plotly.graph_objects as go
+import numpy as np
+
+np.random.seed(42)
+
+# declaring size of arr
+size = 7
+
+x = [f'Product {i}' for i in range(size)]
+y = np.random.randint(low = 0, high = 100, size=size)
+
+# creating the Bar Chart
+fig = go.Figure(go.Bar( 
```

Comment:
These steps will make the graph more appealing:
- add attribute text = y so that values are displayed on the top of bar chart
- Remove the background grid.
- Remove the y axis labels and title